tests/python/relax/test_transform_normalize_global_var.py

```diff
@@ -15,31 +15,31 @@
 # specific language governing permissions and limitations
 # under the License.
 import pytest
+import numpy as np
 
 import tvm
 import tvm.testing
 from tvm import relax
-from tvm import tir
 from tvm.ir.base import assert_structural_equal
 
 import tvm.script
 from tvm.script import tir as T, relax as R, ir as I
 
 
-@pytest.mark.skip_well_formed_check_before_transform
 def test_normalize_relax_function():
-    @I.ir_module
-    class Before:
-        @R.function(private=True)
-        def f():
-            return R.const(1, "int32")
-
-        @R.function
-        def f1():
-            R.func_attr({"global_symbol": "f"})
-            cls = Before
-            gv: R.Tensor((), dtype="int32") = cls.f()
-            return gv
+    # parser will check well-formedness so we can't use it to construct this example
```

Comment:
Instead of changing to the `relax.BlockBuilder`, can we use the `check_well_formed=False` argument?

Reply:
I wish (see above comments) :cry: